app/scripts/lib/channels/duplex.js

```diff
@@ -155,7 +155,7 @@ define([
      * @param {Object} [data]
      * @return {String}
      */
-    createMessageId: function (command, data) { //jshint ignore: line
+    createMessageId: function (command, data) {
```

Comment:
@shane-tomlinson Yeah it checks for unused variables but won't count jsdoc usage as valid https://github.com/eslint/eslint/issues/2813